packages/flutter_tools/lib/src/build_runner/build_runner.dart

```diff
@@ -92,6 +87,8 @@ class BuildRunner extends CodeGenerator {
         }
       }
       stringBuffer.writeln('  build_runner: ^$kMinimumBuildRunnerVersion');
+      stringBuffer.writeln('dependency_overrides:');
+      stringBuffer.writeln('    analyzer: 0.35.3');
```

Comment:
Maybe make the hard-coded analyzer version into a named constant next to `kMinimumBuildRunnerVersion`?